README.md

```diff
@@ -140,6 +149,7 @@ The repository includes these files:
 | [`.github/workflows/ci.yml`](###GitHub-Actions) | The [`ci.yml`](.github/workflows/ci.yml) file defines the pipeline that runs on [GitHub Actions](https://github.com/matlab-actions/overview) |
 | [`Jenkinsfile`](###Jenkins) | The [`Jenkinsfile`](Jenkinsfile) file defines the pipeline that runs on [Jenkins](https://plugins.jenkins.io/matlab/) |
 | [`.travis.yml`](###Travis-CI) | The [`.travis.yml`](.travis.yml) file defines the pipeline that runs on [Travis CI](https://docs.travis-ci.com/user/languages/matlab/) |
+| [`.gitlab.yml`](###GitLab-CI/CD) | The [`.gitlab.yml`](.gitlab.yml) file defines the pipeline that runs on [GitLab CI/CD](https://docs.gitlab.com/ee/ci/) |
```

Comment:
Doesn't this need to be `.gitlab-ci.yml` instead of `.gitlab.yml`? I've only seen it referenced as `.gitlab-ci.yml` in the GitLab CI doc.

```suggestion
| [`.gitlab-ci.yml`](###GitLab-CI/CD) | The [`.gitlab-ci.yml`](.gitlab-ci.yml) file defines the pipeline that runs on [GitLab CI/CD](https://docs.gitlab.com/ee/ci/) |
```

Reply:
`.gitlab-ci.yml` should be better, let's follow the official documents.